src/clarity-angular/wizard/wizard-page.ts

```diff
@@ -171,12 +171,12 @@ export class WizardPage implements OnInit {
 
     // If our host has an ID attribute, we use this instead of our index.
     @Input("id")
-    _id: string = (wizardPageIndex++).toString();
+    _id: string = `${wizardPageIndex++}`;
 
     public get id() {
-        if (!this._id) {
+        if (this._id == null) {
```

Comment:
Hi @imcotton 
First, our linting calls for triple equals. So I'm not sure how this passes.
Second, this check would allow values of empty string and false to pass through. But that does give me an idea for more unit tests on the other PR to guard against those.